props/configs/src/main/resources/dh-defaults.prop

```diff
@@ -71,3 +71,47 @@ client.configuration.list=java.version,deephaven.version,barrage.version,groovy.
 # jar, and a class that is found in that jar. Any such keys will be made available to the client.configuration.list
 # as <key>.version.
 client.version.list=deephaven=io.deephaven.engine.table.Table,barrage=io.deephaven.barrage.flatbuf.BarrageMessageWrapper,groovy=groovy.lang.GroovyShell
+
+# Disabled by default, applications should enable this after their TLS certificates are
+# deployed and confirmed to work, as disabling this after the fact can result in clients
+# remembering the old value for a long time.
+http.add.header.Strict-Transport-Security.enabled = false
+http.add.header.Strict-Transport-Security.value = max-age=31536000; includeSubDomains
+
+# Disable by default, but can be enabled if iframe widgets will not be used from other origins.
+# This can optionally be fine tuned to allow specific origins or same origin loading.
+http.add.header.X-Frame-Options.enabled = false
+http.add.header.X-Frame-Options.value = DENY
```

Comment:
@mofojed talking to Colin, sounds like this has been explicitly requested by some customers. Looks like it only supports DENY / SAMEORIGIN. This will mean customers who enable this header will not be able to use the VS Code extension. At the very least, we'll need to document it as such in the appropriate config docs but may also be worth documenting as a deprecated not recommended setting to enable and encourage users to use the more flexible CSP options.